stations.csv

```diff
@@ -4470,7 +4470,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 5851;Vesoul;vesoul;8718500;87185009;47.6177734;6.151926;;FR;Europe/Paris;f;f;f;t;f;f;FRXVO;VES;t;;f;8700385;f;;f;;f;;f;;f;;f;;;;f;;f;8700947;f;;f;FRXVO;f;t;;;;;;;;;;;;ヴズー;브줄;;;Везуль;;;沃苏勒
 5852;St-Michel—Valloire;st-michel-valloire;8774236;87742361;45.216916;6.4719135;;FR;Europe/Paris;f;f;f;t;f;f;FRXVR;SMI;t;;f;8702381;f;;f;;f;;f;;f;;f;;;;f;;f;8700263;f;;f;FRXVR;f;f;;;;;;;;;;;;;;;;;;;
 5853;Valenciennes;valenciennes;8734300;87343004;50.363259;3.517115;;FR;Europe/Paris;f;f;f;t;f;f;FRXVS;VSN;t;;f;8700078;f;;f;4968;t;;f;;f;;f;;;;f;;f;8700093;f;;f;FRXVS;f;t;;;;;;;;;;;;ヴァランシエンヌ;발랑시엔;;;Валансьен;;;瓦朗謝訥
-5854;Vitré;vitre;8747150;87471508;48.122671;-1.211754;;FR;Europe/Paris;f;f;f;t;f;f;FRXVT;VTE;t;;f;8702235;f;;f;;f;;f;;f;;f;;;;f;;f;8700715;f;;f;FRXVT;f;t;;;;;;;;;;;;ヴィトレ;;;;Витрэ;;;
+5854;Vitré;vitre;8747150;87471508;48.122671;-1.211754;;FR;Europe/Paris;f;f;f;t;f;f;FRXVT;VTE;t;;f;8702235;f;;f;;f;;f;XVT;t;;f;;;;f;;f;8700715;f;;f;FRXVT;f;t;;;;;;;;;;;;ヴィトレ;;;;Витрэ;;;
```

Comment:
Just making sure, is it normal that we have FRXVT, VTE and XVT?

Reply:
I don't know for FRXVT and VTE; I just added XVT, and I am sure it's the Ouigo code for Vitré.